container/images/base/Dockerfile

```diff
@@ -1,14 +1,23 @@
-FROM java:8u111-jdk
-RUN update-alternatives --set java /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java
+FROM python:3.7
```

Comment:
Why do we need Python in the base image?

Reply:
Install Python 3.7 is fiddly and very time consuming, besides installed Java is fairly easy, just coping the layers or folders that relate to the home folder and a couple of other small settings to the environment works just out of the box.

Reply:
It does mean adjusting a couple of dependent languages, actually the last one left is Scala - but that also means we have upgraded Ubuntu on the base image from its predecssor to `jessie`.

Reply:
In my view this is slightly more efficient but if you have a better reason to install python 3.7 via the traditional methods I'm happy to do that, not sure if would deliver anything better.